src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/markup/ListItem.java

```diff
@@ -26,6 +26,6 @@ public ListItem(Node... children) {
 
     @Override
     boolean isValidChild(Node child) {
-        return child instanceof Text || child instanceof Image;
+        return child instanceof Text || child instanceof Image|| child instanceof Link;
```

Comment:
Please check the unit test for this class, there probably needs to be another case added to cover the new condition